docs/docs-beta/docs/dagster-plus/deployment/hybrid/agents/kubernetes.md

```diff
@@ -64,11 +68,32 @@ helm --namespace dagster-cloud upgrade agent \
     --values ./values.yaml
 ```
 
+## Troubleshooting tips
+
+You can see basic health information about your agent in the Dagster+ UI:
+
+[comment]: <> (TODO: Screenshot of Dagster+ Deployments agents tab)
+
+### View logs
+
+```shell
+kubectl --namespace dagster-cloud logs -l deployment=agent
```

Comment:
i'm sure it works, but I've never seen putting --namespace between kubectl and the command, is that common?